generated/nidigitalpattern/nidigitalpattern.proto

```diff
@@ -21,29 +21,17 @@ service NiDigital {
   rpc ApplyLevelsAndTiming(ApplyLevelsAndTimingRequest) returns (ApplyLevelsAndTimingResponse);
   rpc ApplyTDROffsets(ApplyTDROffsetsRequest) returns (ApplyTDROffsetsResponse);
   rpc BurstPattern(BurstPatternRequest) returns (BurstPatternResponse);
-  rpc BurstPatternSynchronized(BurstPatternSynchronizedRequest) returns (BurstPatternSynchronizedResponse);
```

Comment:
I think as a first pass getting all these methods here would clear up what else is missing / different.